storage/database/dynamodb.go

```diff
@@ -97,20 +101,27 @@ func createTestDynamoDBConfig() *DynamoDBConfig {
 		Region:             "ap-northeast-2",
 		Endpoint:           "https://dynamodb.ap-northeast-2.amazonaws.com",
 		TableName:          "dynamo-test-tmp",
-		ReadCapacityUnits:  dynamoReadCapacityUnits,
-		WriteCapacityUnits: dynamoWriteCapacityUnits,
+		IsProvisioned:      false,
+		ReadCapacityUnits:  10000,
+		WriteCapacityUnits: 10000,
```

Comment:
How about declaring a global variable called `DefaultDynamoDBConfig` and then use it in `cmd/utils/flags.go` as default values ?

Reply:
review reflected :)